docs/music.rst

```diff
@@ -3,13 +3,15 @@ Music
 
 .. py:module:: music
 
-This is the ``music`` module. You can use it to play simple tunes, provided
-that you connect a speaker to your board. By default the ``music`` module
-expects the speaker to be connected via pin 0:
+This is the ``music`` module and you can use it to create and play melodies.
+By default sound output will be via the built-in speaker **V2** and via the 
+edge connector on pin 0. You can connect a wired headphones or a speaker to
+pin 0 and GND on the edge connector to hear the sound:
 
 .. image:: music-pins.png
 
-This arrangement can be overridden (as discussed below).
+You can change the default pin configuration to disable the built-in speaker
```

Comment:
Technically that wouldn't disable the speaker, just not output the sound to it. Disabling it goes via pin_speaker.disable().

Reply:
Updated in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5